src/lib/y2network/sysconfig/interface_file.rb

```diff
@@ -336,7 +405,7 @@ def fetch_collection(key, type)
       end
 
       def collection_keys(key)
-        collection_keys = defined_variables.select do |k|
+        collection_keys = (defined_variables || []).select do |k|
```

Comment:
`#defined_variables` should return always an array.

Reply:
There was some error when checking readers but it has been already addressed. So will report in case of find again.